src/auth-service/utils/request.js

```diff
@@ -106,9 +106,7 @@ const request = {
               data: createdCandidate,
               status,
             });
-          }
-
-          if (responseFromSendEmail.success === false) {
+          } else if (responseFromSendEmail.success === false) {
```

Comment:
addressing a potential runtime error and sending HEADERS after they are already sent.